crypto/curve25519/curve25519.c

```diff
@@ -1874,10 +1874,13 @@ static void sc_muladd(uint8_t *s, const uint8_t *a, const uint8_t *b,
   s[31] = s11 >> 17;
 }
 
-void ED25519_keypair(uint8_t out_public_key[32], uint8_t out_private_key[64]) {
-  uint8_t seed[32];
-  RAND_bytes(seed, 32);
+#define ED25519_SEED_LEN 32
```

Comment:
Should this be in the header file alongside the other related constants? https://github.com/awslabs/aws-lc/blob/736fce4b11417ea07ae0df06913d7e87a3cd58ca/include/openssl/curve25519.h#L71

Reply:
That's better.